src/Microsoft.Azure.SignalR/HubHost/ServiceHubDispatcher.cs

```diff
@@ -27,7 +27,7 @@ internal class ServiceHubDispatcher<THub> : IConnectionFactory where THub : Hub
         private readonly IClientConnectionFactory _clientConnectionFactory;
         private readonly string _userId;
         private static readonly string Name = $"ServiceHubDispatcher<{typeof(THub).FullName}>";
-        private static Dictionary<string, string> CustomHeader = new Dictionary<string, string> { { "User-Agent", ProductInfo.GetProductInfo() } };
+        private static Dictionary<string, string> CustomHeader = new Dictionary<string, string> { { "Asrs-UserAgent", ProductInfo.GetProductInfo() } };
```

Comment:
Let's reference #198 here and add comment to explain why we are using a custom header.
Also it helps to include a link to the [source](https://referencesource.microsoft.com/#system/net/System/Net/_HeaderInfoTable.cs,92). Because we don't have test to cover this use case for now.